controller/search_blackbox_test.go

```diff
@@ -63,22 +101,18 @@ func TestSearch(t *testing.T) {
 		},
 		uuid.NewV4())
 	require.Nil(t, err)
-	controller := NewSearchController(service, gormapplication.NewGormDB(DB), spaceBlackBoxTestConfiguration)
+	controller := NewSearchController(s.service, gormapplication.NewGormDB(s.DB), s.spaceBlackBoxTestConfiguration)
 	q := "specialwordforsearch"
 	_, sr := test.ShowSearchOK(t, nil, nil, controller, nil, nil, q)
 	require.NotEmpty(t, sr.Data)
 	r := sr.Data[0]
 	assert.Equal(t, "specialwordforsearch", r.Attributes[workitem.SystemTitle])
 }
 
-func TestSearchPagination(t *testing.T) {
-	resource.Require(t, resource.Database)
-	defer cleaner.DeleteCreatedEntities(DB)()
-	service := getServiceAsUser()
-
-	wiRepo := workitem.NewWorkItemRepository(DB)
+func (s *searchBlackBoxTest) TestSearchPagination() {
+	t := s.T()
```

Comment:
@xcoulon I've seen that you do this and it is fine. But I haven't written this test. I just re-worked it. About the `s.T()` I just didn't want to change much of the existing code or at least keep the changes to a minimum. That's why I've done `t := s.T()`.